### PR TITLE
To add domains allowed additionally to default ones.

### PR DESCRIPTION
**Pull request by dkashyn-sfdc**
Update to latest buildpack.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,10 @@
-FROM heroku/heroku:16-build as base
+FROM heroku/heroku:18-build as base
 
-ENV PHP_BUILDPACK_VERSION v148
+ENV PHP_BUILDPACK_VERSION v174
```
Comment (dkashyn-sfdc) [inline Dockerfile:3]: old one complained too much and not sure it even worked properly due EOL of PHP version in 148

File: workbench/config/defaults.php [hunk 2]
```diff
@@ -1006,4 +1006,12 @@
         "overrideable" => false,
         "dataType" => "string"
     );
+
+    $config["additionalDomainWhitelist"] = array(
+        "label" => "Additional whitelisted domains",
+        "description" => "To add extra domains to whitelist",
+        "default" => array(),
+        "overrideable" => true,
```
Comment (dkashyn-sfdc) [inline workbench/config/defaults.php:1014]: to fully sure about this one

Comment (jamangini) [reply workbench/config/defaults.php:1014]: missing a semi-colon ?

**Comment by dkashyn-sfdc** [comment]
@ntotten could you please review?

**Pull request closed without merging**